### PR TITLE
fix: make EmptyState implementation consistent with Vanilla Framework

### PR DESCRIPTION
**Pull request by vladimir-cucu**
## Done

- Made `EmptyState` component design be consistent with [No content Empty State](https://vanillaframework.io/docs/patterns/empty-state#no-content) in Vanilla Framework.

## Fixes

- #1071 

## Screenshots

Screenshot from 2024-04-26 23-59-55

### CODE REVIEW

**Comment by vladimir-cucu** [comment]
Checked the changes in [percy](https://percy.io/bb49709b/react-components/builds/33913494) and am not sure why some of them are there. Checked locally the storybook differences between main and my branch, and couldn't replicate the changes spotted by percy, apart from the 2 changes related to `EmptyState` component.

**Comment by bartaz** [comment]
> Checked the changes in [percy](https://percy.io/bb49709b/react-components/builds/33913494) and am not sure why some of them are there. Checked locally the storybook differences between main and my branch, and couldn't replicate the changes spotted by percy, apart from the 2 changes related to `EmptyState` component.

Yes, it looks like the diff is made by older version of main branch for some reason. I'll have a look if I can restart those to get them cleaned up.

**Comment by bartaz** [comment]
@vladimir-cucu I think the Percy issue is caused by the fact that your branch is likely not created from freshest `main`, so you are couple commits behind, making your snapshots show older versions of some components that changed in `main` branch.

Could you fetch and rebase on top of fresh `main` and push again? This should hopefully fix that.

**Review by bartaz — APPROVED** [review]
LGTM, thanks!

File: src/components/EmptyState/EmptyState.stories.mdx [hunk 1]
```diff
@@ -54,8 +54,8 @@ export const doNothing = () => {};
             name="containers"
             style={{
               opacity: 0.25,
-              height: "2.5rem",
-              width: "2.5rem",
+              height: "10rem",
+              width: "10rem",
```
Comment (bartaz) [inline src/components/EmptyState/EmptyState.stories.mdx:58]: Icons shouldn't be used this way. They were not designed to be scaled that much up (and weird things may start happening).

This one for example doesn't scale on small screens due to missing max-width that we have on `img`:
image

Ideally, this should only be used with images. Would be good not to document an example with a bad pattern like this.

Maybe let's use this one from assets: https://assets.ubuntu.com/v1/c17e0d92-container.svg

**Comment by vladimir-cucu** [comment]
> @vladimir-cucu I think the Percy issue is caused by the fact that your branch is likely not created from freshest `main`, so you are couple commits behind, making your snapshots show older versions of some components that changed in `main` branch.
> 
> Could you fetch and rebase on top of fresh `main` and push again? This should hopefully fix that.

@bartaz After rebase on top of `main` with latest changes and 2 more placeholder PRs to restart percy, it finally shows the expected changes!

**Review by bartaz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**